wiremock/src/main/content/jcr_root/var/stubs/wiremock/samples/hello-world.stub.groovy

```diff
@@ -13,6 +13,11 @@ stubs.server.with {
     stubFor(get("/json")
             .willReturn(okJson("{ \"message\": \"Hello\" }")))
 
+    stubFor(get("/image")
+            .willReturn(aResponse()
+                    .withBodyFile("samples/logo.png")
+                    .withHeader("Content-Type", "image/png")))
```

Comment:
What if header is not specified? Wiremock has no auto detection feature? Tika based?

Reply:
I am not aware of auto-detection